packages/web-components/src/components/va-breadcrumbs/va-breadcrumbs.tsx

```diff
@@ -25,7 +28,22 @@ export class VaBreadcrumbs {
    * Adds an aria-label attribute to the <nav /> element.
    */
   @Prop() label?: string = 'Breadcrumb';
+  /**
+     * Whether or not the component will use USWDS v3 styling.
+     */
+  @Prop() uswds?: boolean = false;
+  /**
+     * Whether or not the component will wrap the breadcrumbs. This prop is available when `uswds` is set to `true`.
+     */
+  @Prop() wrapping?: boolean = false;
+
+  @Prop() breadcrumbList?: string;
```

Comment:
This prop needs documentation.  
<img width="1052" alt="Screenshot 2023-07-06 at 3 36 31 PM" src="https://github.com/department-of-veterans-affairs/component-library/assets/872479/43fc4f2d-06ef-40e8-99c3-836137230895">